go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/opencontainers/runc
 
-go 1.21
+go 1.22
```

Comment:
Should we keep this 1.22.4,  since 1.22.0-3 has known issue with runc?

Reply:
We want to test with the latest 1.22, not just 1.22.4. That will just test 1.22.4, right?

Sure, we don't want to test with earlier than 1.22.4, but IIUC there is no way to express that and no CI is using failing due to that (all are using older already). IMHO we should keep 1.22 to test the latest releases too.

Reply:
Right, alas there's no way to say "we want 1.22 >= 1.22.4" (I tried hard to find it but failed).